.github/workflows/ProductionRelease.yml

```diff
@@ -0,0 +1,68 @@
+name: Create Release
+
+on:
+  push:
+    branches: [ "prod" ]
+
+jobs:
+  Build_Release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Master
+      uses: actions/checkout@v3
+      with:
+        submodules: true
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v2
+      with:
+        dotnet-version: 6.0.x
+    - name: Restore dependencies
+      run: dotnet restore
+    - name: Build
+      run: dotnet build --no-restore
+    - name: Test Build
+      run: dotnet test --no-build --verbosity normal
+    - name: Setup build artifacts
+      run: |
+        mkdir Artifacts
+        mv ValheimExportHelper/bin/Debug/net6.0/ValheimExportHelper.dll  Artifacts/ValheimExportHelper.dll
+        mv ValheimExportHelper/bin/Debug/net6.0/YamlDotNet.dll  Artifacts/YamlDotNet.dll 
```

Comment:
We should build Release and not Debug right?

Reply:
Whoops! good catch